imports/ui/Footer/index.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
```

Comment:
We won't need `, { Component }` when Footer is written as a pure function (see below)